README.md

```diff
@@ -28,11 +30,11 @@ Heroku Connect Toolbelt Plugin
 
 Create a new mapping
```

Comment:
Ok, will do. Can we keep the connect:mapping:create command in the "Now sync the Contact table" step below? The tutorial would feel incomplete without it.